flocker/node/agents/cinder.py

```diff
@@ -70,6 +73,8 @@ def _run_with_logging(self, *args, **kwargs):
         with OPENSTACK_ACTION(operation=[method_name, args, kwargs]):
             try:
                 return method(*args, **kwargs)
+            except KeystoneOverLimit as e:
+                raise
```

Comment:
I don't understand this bit. 
Don't we want a new Message to be logged here? 
Otherwise we won't see the `retryAfter` details.
There doesn't seem any point in catching and then re-raising it.